### DISCUSSION TITLE
Where should the Vox Pupuli sync agenda live?

### DISCUSSION THREAD

**Discussion by binford2k** [puppetlabs/community#53] [Polls]
Vox Pupuli leadership meets once a month with the Puppet Community team. We share ideas, plan roadmaps, share challenges, make decisions, etc. It's open to the public, and you're invited any time you'd like to see the sausage being made. You can find the information on the [Community Calendar](https://puppet.com/community/calendar).

Currently, the [agenda and notes live in a Google doc](https://docs.google.com/document/d/1rD1-HC36qNZL5FXEeSE1rtTjMHM45O6ylk3xwBmZDFg/edit#), but that's on borrowed time. In the post-acquisition integration, we're moving Puppet's Google workspace into Microsoft 365 and we won't have the ability to share docs publicly to be collaboratively edited. So I'd like to find a new home for it.

Here are some of the things that I personally find important to varying degrees:
- publicly viewable
- can be edited by anyone, with moderation for untrusted users
- doesn't require yet-another-login
- collaborative real-time editing
- revision history so we can undo mistakes

What's important to you? Do you have suggestions? Put them in comments, or vote if you're happy with one of these options.

UPDATE: we are now using https://github.com/voxpupuli/community-triage/wiki
Update 2: moved to https://github.com/orgs/voxpupuli/projects/10/views/1 as the wiki was tedious

**Comment by bastelfreak** [comment]
back in the old old days, when the Puppet core team or the modules team did public reviews, we had notes in a git repo. I think thats the way to go. easy to edit, GitHub can render it to markdown, everybody can view them, we have access control and revision control. I currently don't have an opinion on a simple git repo vs GitHub wiki vs github pages.

Edit: It was https://github.com/voxpupuli/community-triage

**Reply by binford2k** [reply]
that's right, I'd forgotten the name of that one and couldn't find it. I thought it was in our namespace 🤣 

I updated the poll

**Reply by ekohl** [reply]
And for people like me who like to click links: https://github.com/voxpupuli/community-triage/wiki contains the agenda, at least for the 2024-01-09 meeting.

**Comment by smortex** [comment]
IMHO wiki pages are okay. I don't think it allows concurrent edits but that is probably not a major issue, and it's available as a git repo to clone if we want to get all content at once.

GitHub Projects could also maybe be nice to track things over time (e.g. What the state of Debian 12 packages?), but I am pretty sure random GitHub users (i.e. non-voxpupuli members) would only have a read-only access.  Maybe it's not an issue, and these external users can add something they have in there mind in a wiki page, and if during the meeting we realize we have to think about it and re-talk about it in the next meeting we can add a card for it in the project.

**Reply by davidsandilands** [reply]
I was pretty much going to say the same, some experimentation with GitHub projects may be worthwhile 

This seemed quite an organised way of doing it https://github.com/tronprotocol/pm?tab=readme-ov-file @binford2k

**Comment by binford2k** [comment]
I suppose that I should mark this as resolved. Agenda is at https://github.com/orgs/voxpupuli/projects/10/views/1 now.
